src/app/pages/games/game-card/game-card.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { fromEvent, Observable } from 'rxjs';
+import { Blog } from 'src/app/domain/blog.model';
+
+@Component({
+  selector: 'game-card',
+  templateUrl: './game-card.component.html',
+  styleUrls: ['./game-card.component.css']
+})
+export class GameCardComponent implements OnInit {
+  @Input() poBlogData: Blog | undefined;
+
+  gameButtonDesc: string | undefined;
+  gameImageAltText: string | undefined;
+
+  isMobile: boolean | undefined = false;
```

Comment:
I think you don't need undefined type here cos you have a default boolean value set.